### ISSUE TITLE
Collection of datasets of at least 1000 entries of features and lables in csv or json. Ex - {"word": "love", "mood": "romantic"} or {"sentence": "Hey, Babe", "mood": "romantic"}

### ISSUE THREAD

**Issue by Ayush-projects** [Ayush-projects/chords#30]

**Comment by nikhil-Vaibhav** [comment]
Hey @Ayush-projects I would like to start with this part of project. Please assign me this task and help me a little bit of what features and labels to be added.

**Comment by Sushi22** [comment]
@Ayush-projects it is kind of emotion detection from text . Can we go with some publicly available datasets?

**Comment by Ayush-projects** [comment]
Yes, we can @Sushi22 @nikhil-Vaibhav, I am planning to do this using natural language processing. Anybody who is willing to work on the machine learning part can comment on the newly created issue. We will arrange a gmeet session very soon.
